tests/commands/test-kob-help.sh

```diff
@@ -1,21 +1,57 @@
 #!/bin/bash
-# check if kob is installed
-# Remove any old output.txt files  
-#rm output.txt
-# run the command
-kob help > output.txt
-# check the output of the command
-# if ( cat output.txt | grep "known pattern") 
- echo " Success"
-#  return 
- else 
-  echo "Failed $0"
-# check and remove
-rm output.txt
-
-
 
+function __test_kob_init
+{
+	commands="install uninstall status list version help update upgrade"
```

Comment:
@asa1997   this will work,, however  see if  the grep output can be compared rather an running a loop of commands in the validate , this will help tests fail fast.